.github/workflows/tests.yaml

```diff
@@ -78,11 +78,11 @@ jobs:
 
       - name: Install Hadolint binary (github)
         run: |
-          mkdir hadolint-bin
+          mkdir -p $HOME/.local/bin
+          echo "$HOME/.local/bin" >> $GITHUB_PATH
           curl -sL -o hadolint https://github.com/hadolint/hadolint/releases/download/v2.6.0/hadolint-$(uname -s)-$(uname -m)
           chmod +x hadolint
-          mv hadolint hadolint-bin/.
-          echo "$GITHUB_WORKSPACE/hadolint-bin" >> $GITHUB_PATH
+          mv hadolint $HOME/.local/bin/
```

Comment:
these changes aren't strictly needed right? do you think it's better to use $HOME or $GITHUB_WORKSPACE?

Reply:
I am following the example from [GitHub docs](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/workflow-commands-for-github-actions#example-of-adding-a-system-path).